### PR TITLE
Docs for NFT and Swaps

### PR DESCRIPTION
**Pull request by MaximFischuk**

### CODE REVIEW

File: docs/api/sdk/java/tutorial.md [hunk 1]
```diff
@@ -261,56 +279,18 @@ TransactionFeeRequest feeRequest = TransactionFeeRequest.builder()
                                 .tokenIdentifier(Token.createETH())
                                 .build();
 TransactionFeeDetails fee = wallet.getProvider().getTransactionFee(feeRequest);
-String txHash = wallet.syncTransfer(
```
Comment (StanislavBreadless) [inline docs/api/sdk/java/tutorial.md:264]: why not showing the returned value?

File: docs/api/sdk/java/tutorial.md [hunk 2]
```diff
@@ -332,9 +312,243 @@ wallet.syncWithdraw(
...
+
+#### Creating limit order
+
+Limit orders provide a way to exchange a certain token for another at a certain price. They are designed to be used primarily by other platforms that want to provide trustless and scalable exchange services.
+
+The differences between an atomic swap and a limit order are:
+
+- limit orders infer the amount that can be exchanged directly from the balance
+- limit orders can be partially filled
+- limit orders do not increment account's nonce when partially filled
```
Comment (StanislavBreadless) [inline docs/api/sdk/java/tutorial.md]: @ly0va 

Do they increment nonce when they are fully filled?

**Review by StanislavBreadless — COMMENTED** [review]
nice, a few questions

File: docs/api/sdk/java/tutorial.md (hunk 2 above)
Comment (ly0va) [reply docs/api/sdk/java/tutorial.md]: No, limit orders never increment nonce

Comment (ly0va) [reply docs/api/sdk/java/tutorial.md]: Full filling is a special case of partial filling :D

**Review by ly0va — APPROVED** [review]
Please make sure CI passes

**Comment by StanislavBreadless** [comment]
> Please make sure CI passes

It's ok. The CI fails on Firebase preview deployment which can not be done during PRs from forks

**Review by StanislavBreadless — APPROVED** [review]

**Comment by StanislavBreadless** [comment]
@MaximFischuk GitHub wouldn't allow me to merge the PR because of the ci. I'll have to re-open it from my account

**Pull request merged**